oat-sa/extension-tao-itemqti
# Fix/AUT-1279/style editor clear cache in item authoring

**Backport to [138 sprint](https://github.com/oat-sa/tao-community/tree/v0.138.5-alpha)**

Related to: https://oat-sa.atlassian.net/browse/AUT-1279
https://oat-sa.atlassian.net/browse/AUT-1281

Add `clearCache` after exit from itemAuthoring.

**Preconditions:**
Two Items created

1. with a custom font set
2. without any custom styles/fonts

**Steps to test**

1. Refresh the Items page
2. Open Authoring mode of the item without any custom fonts in order to make sure it has a default font
3. Close the Authoring
4. Open the item with a custom font for Authoring
5. Close the Authoring
6. Open the item from the step 2 for Authoring again
7. Save the item (making absolutely no changes to it)
8. Open the item’s preview from Authoring mode OR Refresh the page and open the item’s authoring again
9. No custom fonts are applied to the previewed item as no changes were made to it
